orc8r/cloud/helm/orc8rlib/templates/_helpers.tpl

```diff
@@ -0,0 +1,28 @@
+{{/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/}}
+{{/* Generate basic labels */}}
+{{- define "labels" -}}
+app.kubernetes.io/name: {{ .Chart.Name }}
+app.kubernetes.io/component: {{ .Chart.Name }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/managed-by: helm
+app.kubernetes.io/part-of: orc8r
```

Comment:
This is needed for the service registry so that we can query all or8cr components. In the stacked PR which adds services to the base orc8r helm chart, I will change this label as well.